_posts/2025-06-25-mast-heatmap.md

```diff
@@ -0,0 +1,117 @@
+
+Ecliptic coordinates are useful for space-based telescopes, since the sun is — by definition —always at a declination of 0º. In fact, the [survey design of the TESS spacecraft](https://youtu.be/Q4KjvPIbgMI&t=115) takes advantage of this by always keeping one camera on the ecliptic pole, with the others facing opposite the sun.
+
+![the MAST heatmap, reprojected to the plane of the ecliptic](assets/img/posts/20250625/mast_wallpaper_ecl_blue.png)
+<small>Ohhhh, so that's why those observations make a straight line around the plane of the ecliptic! They're from TESS.</small>
+#### Projection 2: Galactic
+Galactic coordinates are, somewhat counter-intuitively, still centered on our Sun. In this case, 0º longitude is defined to be the line between the sun and the center of the galaxy. Latitude measures the angle of an object relative to the flat disk of the galaxy, as seen from Earth.
+
+![](assets/img/posts/20250625/mw.jpg)
+<small>By NASA/JPL-Caltech/ESO/R. Hurt - <a rel="nofollow" class="external free" href="http://www.eso.org/public/images/eso1339e/">http://www.eso.org/public/images/eso1339e/</a>, Public Domain, <a href="https://commons.wikimedia.org/w/index.php?curid=28274906">Source</a></small>
+
+The nice thing about using the galactic coordinate system is that most stars are near the plane of the galaxy, so it shows up quite nicely on our plot!
+![the mast heatmap, projected to galactic coordinates](assets/img/posts/20250625/mast_wallpaper_gal_blue.png)
+<small>The Milky Way, as seen by MAST. Come on, TESS! Observe that last little part in the middle so we can see the whole galaxy.</small>
+## Conclusions and Other Thoughts
+It's pretty dang cool that through metadata alone, we can reconstruct the shape and even texture of the milky way. These images, which are gorgeous, are also a powerful reminder that MAST has a lot of data. And this is a reminder that those data are free for anyone to access, so check them out through our [search forms](https://mast.stsci.edu/search/ui) or our [astroquery API](https://spacetelescope.github.io/mast_notebooks/intro.html)!
+
+If you've read this far, thanks. Here are a few extra tidbits that you might be thinking about right now:
+### Hey, what is that thing I see?
+Great question. Take a look at our annotated map, which highlights a few points of interest.
+![an annotated version of the heatmap](assets/img/posts/20250625/annotated.png)
+
+### I want to make my own map!
+You probably should not do this! It's a pretty large database query, and will degrade the performance of our server if a bunch of people attempt it.
+
+If you absolutely MUST make your own version, please use this [TAR file containing all CAOM observations](https://stsci.box.com/s/ok36t5mmczw6jjj4d6kcmsiiur6vmh1m). Pandas will know how to read it, with a quick `pd.read_pickle()`.
```

Comment:
I was under the impression that pickle was not recommended as it's possible to do malicious code with it. Can we use a more standard format instead? A CSV? HDF5?

It may also be good to describe a bit more what this contains. I assume it's only ra/dec per observation and one observation per row?

Also, would be good to explicitly state the filesize they would be getting if they download this.